### PR TITLE
moved and edited old git info from adding-git-guide

### PR DESCRIPTION
**Pull request by senkevinli**
Moved and reorganized Bradley's old Git guide info. Wasn't sure how to preserve Git Blame/authorship from the adding-git-guide branch, if anyone has an idea on how, please let me know.

### CODE REVIEW

File: guides/installation/src/git/git.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+In most UAV Austin repositories, the main state of the code is in the *master*
+branch. For each card on Trello, generally, we will have a *feature branch* for
+it while it is in development. When it's just about finished, you'll add a
+*pull request* (or *PR*) to merge it into the *master* branch. A team lead will
+review the code real fast, make sure it's good, and then merge it in.
+
+**Merging**:
+Make sure to use the `--no-ff` flag when merging by the command line, this will
+preserve the branch history on the graph.
```
Comment (alexwitt23) [inline guides/installation/src/git/git.md]: I've never used this. Is this default? Maybe add an example here

File: guides/installation/src/git/git.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+## Version Tagging
+
+*Directed towards team leads.*
```
Comment (alexwitt23) [inline guides/installation/src/git/git.md]: We might want to have a link to a doc we all agree on; maybe [this](https://semver.org/)

File: guides/installation/src/git/github.md [hunk 3]
```diff
@@ -0,0 +1,228 @@
...
+
+## Contributing to a Repository
+**Starting a Local Repository**
+
+Go to the repository link of the project that you want to work on or create your own. 
+
+Copy the remote URL onto your clipboard:
+<img src="../img/remotes-url.png" alt="remote-url" width="600">
+
+On your local machine and desired directory:
```
Comment (alexwitt23) [inline guides/installation/src/git/github.md]: `s/and desired/and in the desired` for clarity

File: guides/installation/src/git/github.md [hunk 4]
```diff
@@ -0,0 +1,228 @@
...
+these changes, consider the following commands.
+```
+$ git fetch origin
+```
+Fetch will download data from the remote repository, but will not modify anything locally.
+This is a good way to check up on the progress of other collaborators.
+```
+$ git pull origin master
+```
+Pull runs the aforementioned fetch and then merges the changes in the remote repository to your
```
Comment (alexwitt23) [inline guides/installation/src/git/github.md]: `pull` to be clear we are talking about the command

File: guides/installation/src/git/github.md [hunk 5]
```diff
@@ -0,0 +1,228 @@
...
+there are.
+
+
+<img src="https://i.imgur.com/UklMnBr.jpg" alt="Cat Picture" style="width:30%">
+
+We're going to want to make a new branch off of *master* to work on, so we
+don't bother someone else working on the same project.
+
+Feature branches should have a short, descriptive name. Our naming convention
+for feature branches is to simply use lower-case names separated by hyphens.
```
Comment (alexwitt23) [inline guides/installation/src/git/github.md]: I'd prefer to see here something like "Branch naming scheme will depend on the team lead, but here we will simply ...". 
I prefer `initials-feature` to know who is working on what.

File: guides/installation/src/git/github.md [hunk 6]
```diff
@@ -0,0 +1,228 @@
...
+machines and has nothing in it. We'll add something here so this branch has
+meaning and then we'll get this new branch on GitHub.
+
+So in this case, let's say we're editing the file `index.html`. This is where
+we're adding our cat picture.
+
+After we make our change, we'll need to get it committed to our local
+repository and then to the remote repository.
+
+Commit messages should be should and descriptive. It's also better to make
```
Comment (alexwitt23) [inline guides/installation/src/git/github.md]: `s/should be should/should be`

File: guides/installation/src/git/github.md [hunk 7]
```diff
@@ -0,0 +1,228 @@
...
+fairly simple.
+
+In the GitHub UI you can simple switch branches with the dropbox or by clicking
+"x branches" at the top. When on the correct branch, clicking on "x commits"
+will show all the changes made as well.
+
+
+### Continuing our Work
+
+Alright so we can still continuing working in our branch. In this example we'll
```
Comment (alexwitt23) [inline guides/installation/src/git/github.md]: `s/continuing/continue`

File: guides/installation/src/git/github.md [hunk 8]
```diff
@@ -0,0 +1,228 @@
...
+```
+$ git push origin add-cat-picture
+```
+
+Now this is on the remote repository. Note that we don't need the `-u` flag as
+that was just for the first push.
+
+### Making a Pull Request
+
+Alrighty, so we've gotten to the point where we think are changes are nearly
```
Comment (alexwitt23) [inline guides/installation/src/git/github.md]: `s/think are/think our`

**Review by alexwitt23 — COMMENTED** [review]
Thanks for the PR. I've added some comments, open to discussion. To me, it looks like a lot of the same information is in `using-git-and-github.md` and `git.md`

File: guides/installation/src/git/github.md (hunk 5 above)
Comment (senkevinli) [reply guides/installation/src/git/github.md]: Fixed

File: guides/installation/src/git/git.md (hunk 1 above)
Comment (senkevinli) [reply guides/installation/src/git/git.md]: I added some examples in the "Merging" section. Let me know what you think

**Comment by alexwitt23** [comment]
Thanks for addressing feedback! CC @srikarsganti and @chaseguy15 to take a look, too. We should all agree on this -- important docs

File: guides/installation/src/git/git.md [hunk 9]
```diff
@@ -0,0 +1,158 @@
...
+- [Pro Git Book](https://git-scm.com/book/en/v2)
+- [Official Documentation](https://git-scm.com/docs)
+
+### Branching:
+In order for us to all work together on the same project, we use Git branching
+often. This allows us to all contribute at the same time without interference,
+and for us to merge our work into the main line when it's done and has been
+verified.
+
+In most UAV Austin repositories, the main state of the code is in the *master*
```
Comment (srikarsganti) [inline guides/installation/src/git/git.md]: "main state" -> "latest version" for clarity
```suggestion
In most UAV Austin repositories, the latest version of the code is in the *master*
```

File: guides/installation/src/git/git.md [hunk 10]
```diff
@@ -0,0 +1,158 @@
...
+branch. For each card on Trello, generally, we will have a *feature branch* for
+it while it is in development. When it's just about finished, you'll add a
+*pull request* (or *PR*) to merge it into the *master* branch. A team lead will
+review the code real fast, make sure it's good, and then merge it in.
```
Comment (srikarsganti) [inline guides/installation/src/git/git.md]: Potential clarity improvements 
```suggestion
branch. For each card on Trello, generally, we will create a new branch where we can add features, fix bugs, and develop (we call these branches *feature branches*). When it's just about finished, you'll add a
*pull request* (or *PR*) to merge it into the *master* branch. A team lead will
review the code real fast, make sure it's good, and then merge it in.
```

File: guides/installation/src/git/git.md [hunk 11]
```diff
@@ -0,0 +1,158 @@
...
+with the master branch is also the latest commit in the master branch. Fast-forward
+merge is enabled by default.
+
+Here's an example (switch to an appropriate directory):
+```
+$ git init
+$ touch a.txt
+$ echo "commit 1" > a.txt
+$ git add . && git commit -m "first commit"
+$ git checkout -b feature branch
```
Comment (srikarsganti) [inline guides/installation/src/git/git.md]: adding comments to each of these commands may provide clarity. Also, typo on line 77, should be "feature-branch"
```suggestion
$ git init #creates a new repository, starts you off in the master branch
$ touch a.txt #creates a new file
$ echo "commit 1" > a.txt #adds text to a file
$ git add . && git commit -m "first commit" #stages a change from the working directory to the staging area, then captures a "snapshot" of the staging area
$ git checkout -b feature-branch #creates and navigates to a new branch called "feature-branch"
```

**Review by srikarsganti — CHANGES_REQUESTED** [review]
Good changes. I made a couple comments and suggestions. Mostly clarity and readability things. Check em out

File: guides/installation/src/git/git.md (hunk 10 above)
Comment (senkevinli) [reply guides/installation/src/git/git.md]: Fixed

File: guides/installation/src/git/git.md (hunk 11 above)
Comment (senkevinli) [reply guides/installation/src/git/git.md]: Fixed

File: guides/installation/src/git/git.md (hunk 9 above)
Comment (senkevinli) [reply guides/installation/src/git/git.md]: Changed to "latest stable version"

**Review by srikarsganti — APPROVED** [review]
Looks good, thanks for updating

**Review by chaseguy15 — COMMENTED** [review]
Looks great, this will be super useful for new people. Thanks!

Do we want to to enforce some of these standards like branch naming and frequency?

**Comment by senkevinli** [comment]
We can talk about it at the next meeting

**Pull request merged**